app/views/users/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for @user do |f| %>
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+  <%= f.label :password %>
+  <%= f.password_field :password %>
+  <%= f.label :address %>
+  <%= f.text_field :address %>
```

Comment:
I can change the model test and validations to not validate presence of address if that will help.